### PR TITLE
pulling newest Extensions nugets to fix MobileTable 'connection' bug

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

File: sample/MobileTable-Node/function.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
             "type": "mobileTable",
             "direction": "out",
             "name": "item",
-            "tableName": "Item"
+            "tableName": "Item",
+            "connection": "AzureWebJobs_TestMobileUri"
```
Comment (pragnagopa) [inline sample/MobileTable-Node/function.json]: Nit: Update wiki to set AzureWebJobs_TestMobileUri

**Comment by pragnagopa** [comment]
:+1:

**Comment by fabiocav** [comment]
:shipit:

**Comment by brettsam** [comment]
FYI reviewers -- I updated this to include the latest Extensions drop which includes my ApiKey bug fix as well.

**Pull request merged**